### ISSUE TITLE
Personalized evaluation with user as ground truth

### ISSUE THREAD

**Issue by sporreking** [MattiasBeming/LiU-AI-Project-Active-Learning-for-Music#87]
Active Learning is used for training, which would introduce som subjectivity into the model.  At the moment, the model is only evaluated by its ability to perform predictions of the general test labels which were compiled by multiple people. This is of course interesting since it shows how large of an impact the subjectivity of the labeling person has - but it is also of interest to see how well the algorithm is able to predict the labeling person's emotions. Instead of the subjectivity being a problem, it could instead solve the question of preference; different people find different emotions with the same kind of music.

**Instead of using these general test labels as ground truth, the user that was responsible for labeling the test data should be queried for labels again. Then, evaluate how good the trained models are att predicting the emotions of the user who labeled for the training data for a number of test songs. Note that this should be a complemetary part of the program - both types of evaluation are interesting to maintain.**

_In practice, this could represent a music service using feedback from a user as Active Learning input, to then knowingly utilize the propagation of subjectivity throughout a user-specific algorithm to suggest mood-based music._
